cpp/ql/src/experimental/Security/CWE/CWE-078/WordexpTainted.qhelp

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>The code passes user input to <code>wordexp</code>. This leaves the code
+vulnerable to attack by command injection, because <code>wordexp</code> performs command substitution.</p>
```

Comment:
It might be nice to expand on this for those unaware of the feature, maybe something along the lines of:
```
Command substitution is a feature that replaces $(command) or `command` with the
output of the given command, allowing the user to run arbitrary code on the system.
```